### PR TITLE
Inteq aesthetic updates, the jobbening

### PR DESCRIPTION
**Pull request by Ebin-Halcyon**
## About The Pull Request
Another bump in aesthetics for everyone's favorite brown clad, lizard kissing, semi legal activity operating PMCs.
This time, with a bump to their job aesthetics, as me and @Drifter7371 have updated the lore a bit to give them different jobs in the faction! They all still listen to Vanguards either way.

If you'd like to read up on what we've got so far and what future jobs may be coming, a hackMD of their lore is [here!](https://hackmd.io/I92RFAodReCTStozPuvJlQ)

Without further ado, screenshots of said sprites in game.

(I've also gone and updated small bits of their sprites with a bunch of consistent gold color, and updating the combat gloves to be more black and inline with the combat gloves in the worn sprite)

Say hello to the Corpsman, and the Artificer, the medical and engineering titles respectively.
image

As per usual, the non-skirt clothing is digitigrade compatible, currently not Vox compatible, but i'm working on that.

## Why It's Good For The Game

Updates to me and Defender's faction I plan to upkeep as long as I can. :]

## Changelog
:cl:
imageadd: The IRMG have sent in a shipment of clothing for their frontline engineers and medical officers!
/:cl:

### CODE REVIEW

**Comment by Ebin-Halcyon** [comment]
Ough fuck.

**Comment by MarkSuckerberg** [comment]
@triplezeta sprite review pls

**Comment by triplezeta** [comment]
waa

**Review by triplezeta — APPROVED** [review]
activities:bangbang:

**Comment by Apogee-dev** [comment]
You might want to tweak the suit storage permissions on the corpsman jacket to accept medical items instead of sec items, or make it more of a hybrid list (medical items plus handguns?)

**Comment by Fikou** [comment]
are you gonna do the above

**Comment by Ebin-Halcyon** [comment]
> are you gonna do the above

I'll get on it after I get back from my job.

**Comment by Calyxman** [comment]
God please tm

**Review by triplezeta — APPROVED** [review]
activities x2 :bangbang:

File: code/modules/clothing/suits/armor.dm [hunk 1]
```diff
@@ -354,6 +354,7 @@
 	desc = "A shortened brown labcoat with an armor vest under it, for the IRMG's support division medical staff."
 	icon_state = "inteq_labcoat"
 	item_state = "inteq_labcoat"
+	allowed = list(/obj/item/analyzer, /obj/item/stack/medical, /obj/item/reagent_containers/syringe, /obj/item/reagent_containers/hypospray, /obj/item/healthanalyzer, /obj/item/flashlight/pen, /obj/item/reagent_containers/glass/bottle, /obj/item/reagent_containers/glass/beaker, /obj/item/reagent_containers/pill, /obj/item/storage/pill_bottle, /obj/item/melee/classic_baton/telescopic, /obj/item/soap, /obj/item/sensor_device, /obj/item/tank/internals/emergency_oxygen, /obj/item/tank/internals/plasmaman, /obj/item/ammo_box, /obj/item/gun/ballistic, /obj/item/gun/energy, /obj/item/melee/baton)
```
Comment (Fikou) [inline code/modules/clothing/suits/armor.dm]: please format this list to be readable
= list(
      thing1,
      thing2,
)

File: code/modules/clothing/suits/armor.dm [hunk 2]
```diff
@@ -349,6 +349,33 @@
...
+		/obj/item/melee/classic_baton/telescopic,
+		/obj/item/soap,
+		/obj/item/sensor_device,
+		/obj/item/tank/internals/emergency_oxygen,
+		/obj/item/tank/internals/plasmaman,
+		/obj/item/ammo_box,
+		/obj/item/gun/ballistic,
+		/obj/item/gun/energy,
+		/obj/item/melee/baton
+		)
```
Comment (Fikou) [inline code/modules/clothing/suits/armor.dm]: add a trailing comma and unindent the last parenthesis

Comment (Fikou) [reply code/modules/clothing/suits/armor.dm]:
```suggestion
		/obj/item/melee/baton,
	)
```

**Comment by Fikou** [comment]
yumy

**Pull request merged**